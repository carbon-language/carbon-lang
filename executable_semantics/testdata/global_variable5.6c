// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test overshadowing of global variable.

var Int x = 1;

fn identity(Int x) {
  return x;
}

fn main() -> Int {
  return identity(0);
}
