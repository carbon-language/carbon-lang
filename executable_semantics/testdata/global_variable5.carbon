// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test overshadowing of global variable.

var x: i32 = 1;

fn identity(x: i32) {
  return x;
}

fn main() -> i32 {
  return identity(0);
}
