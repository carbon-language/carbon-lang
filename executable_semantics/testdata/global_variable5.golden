10: type error in return
expected: Tuple()
actual: Int
EXIT CODE: 255
